### PR TITLE
Update deps

### PR DESCRIPTION
**Pull request by harti2006**
Lifted dependency versions and fixed the resulting issues

### CODE REVIEW

File: fullstop-plugins/fullstop-ami-plugin/src/test/java/org/zalando/stups/fullstop/plugin/ami/config/AmiPluginAutoConfigurationTest.java [hunk 1]
```diff
@@ -4,20 +4,20 @@
...
 import org.zalando.stups.fullstop.plugin.EC2InstanceContextProvider;
 import org.zalando.stups.fullstop.plugin.ami.AmiPlugin;
 import org.zalando.stups.fullstop.violation.ViolationSink;
 
-import static org.assertj.core.api.Assertions.assertThat;
 import static org.mockito.Mockito.mock;
+import static org.assertj.core.api.Assertions.assertThat;
 
 
 @RunWith(SpringJUnit4ClassRunner.class)
```
Comment (ChristianLohmann) [inline fullstop-plugins/fullstop-ami-plugin/src/test/java/org/zalando/stups/fullstop/plugin/ami/config/AmiPluginAutoConfigurationTest.java]: There is an alias class for `SpringJUnit4ClassRunner` called `SpringRunner` now.

File: fullstop-plugins/fullstop-region-plugin/src/test/java/org/zalando/stups/differentnamespace/FullstopApplicationSingleIT.java [hunk 2]
```diff
@@ -20,8 +19,7 @@
 import static org.zalando.stups.fullstop.events.TestCloudTrailEventSerializer.createCloudTrailEvent;
 
 @RunWith(SpringJUnit4ClassRunner.class)
```
Comment (ChristianLohmann) [inline fullstop-plugins/fullstop-region-plugin/src/test/java/org/zalando/stups/differentnamespace/FullstopApplicationSingleIT.java]: `SpringRunner`

**Comment by harti2006** [comment]
:+1:

**Comment by ChristianLohmann** [comment]
👍

**Pull request merged**